src/BenchmarksApps/Mvc/Startup.cs

```diff
@@ -28,6 +35,33 @@ public void ConfigureServices(IServiceCollection services)
             {
                 mvcBuilder.AddNewtonsoftJson();
             }
+
+#if JWTAUTH
+            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme).AddJwtBearer(o =>
+            {
+                o.TokenValidationParameters.ValidateActor = false;
+                o.TokenValidationParameters.ValidateLifetime = true;
+                o.TokenValidationParameters.ValidateAudience = true;
+                o.TokenValidationParameters.ValidateIssuer = true;
+                o.TokenValidationParameters.ValidIssuer = "Test";
+                o.TokenValidationParameters.ValidAudience = "test";
+                o.TokenValidationParameters.IssuerSigningKey = new SymmetricSecurityKey(Convert.FromBase64String("MFswDQYJKoZIhvcNAQEBBQADSgAwRwJAca32BtkpByiveJTwINuEerWBg2kac7sb"));
```

Comment:
Where is this key coming from? Or how do we generate it?

Reply:
The key doesn't matter, I just generated this random one and used a jwt generator to generate the encoding/payload, so its hand crafted to match the Jwt tokens.

Reply:
I will put a comment in here though to describe the process in case we need to generate new claims or want to change the key or something in the future

Reply:
A comment is great, thanks